tests/workbenches/resources.py

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+from __future__ import annotations
+
+from ocp_resources.resource import NamespacedResource
+
+
+class Notebook(NamespacedResource):
+    api_group: str = "kubeflow.org"
```

Comment:
IMHO this should be a class in ocp_resources itself

Reply:
allright, maybe if I start contributing to ocp_resources, I can try to do the work

Reply:
There is a tool that will auto-generate the class for ocp_resources, https://github.com/RedHatQE/openshift-python-wrapper/blob/main/class_generator/README.md
If you need any help with it feel free to ask any of us!

Reply:
* https://github.com/RedHatQE/openshift-python-wrapper/pull/2331